tests/schema/test_generics.py

```diff
@@ -349,46 +349,6 @@ def example(self, info) -> typing.Union[Fallback, Edge[int]]:
     }
 
 
-# def test_supports_generic_in_unions_multiple_vars():
```

Comment:
this test might be useful 🤔

Reply:
😆 Ok, I will restore it.

Reply:
This was commented at that same point. I will try to make it pass now.

Reply:
I managed to make it pass but... why this order matters?
<img width="517" alt="Captura de Tela 2020-09-04 às 13 03 36" src="https://user-images.githubusercontent.com/13003392/92261065-3c810f80-eeaf-11ea-879a-5832fb1628bd.png">

Reply:
good question, it shouldn’t matter! I’ll try to address this in another PR :)

Reply:
I think it is a good idea to open an issue about it, right?